lib/ttnt/storage.rb

```diff
@@ -0,0 +1,67 @@
+module TTNT
+  # A storage to store TTNT data such as test-to-code mapping and metadata.
```

Comment:
"storage" seems redundant here. What do you think about "A utility class to store TTNT data ..." ?